ROLES.md

```diff
@@ -0,0 +1,25 @@
+# Roles
+
+## Contributor
+
+A contributor is any person who has at least 50 contributor 'Activities' in the Contributor Leaderboard for the last 12 months,
+or at least 5 pull requests merged in the last 12 months,
+as shown in https://insights.lfx.linuxfoundation.org/foundation/cncf/overview/github?project=kuadrant&repository=all&routedFrom=Github&dateFilters=Last%2012%20Months&dateRange=2023-08-02%20to%202024-08-01&compare=PP&granularity=month&hideBots=true
+
+## Maintainer
+
+The list of maintainers for the project is at [./MAINTAINERS.md](./MAINTAINERS.md).
+Generally, these people have the ability to create, approve and merge pull requests on any repository.
+
+[TODO: Decide & document rules for how someone can become a maintainer]
+
+### Component specific maintainers
+
+Some components have a subset of maintainers. The list of components and maintainers can be found in the [./MAINTAINERS.md#component-specific-maintainers](./MAINTAINERS.md#component-specific-maintainers) section.
+These people have the same permissions as maintainers.
+However, their role includes having domain expertise in those components.
+They can be thought of as technical leads in a smaller part of the overall project.
+There are no set rules for how a person can move from a maintainer role to a component specific maintainer role.
+Generally, if a person is sufficiently active in a particular area, they may add themselves, be added, or request to be added to the list for a component.
```

Comment:
What about people moving from a contributor role to a component maintainer role?  That is, someone who isn't a general maintainer, but wants to move up to a component maintainer?